frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -677,6 +683,35 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 setScheduledChangeOperation: (_, { changeType }) => changeType,
             },
         ],
+        isRecurring: [
+            false,
+            {
+                setIsRecurring: (_, { isRecurring }) => isRecurring,
+                // Reset when operation changes away from UpdateStatus
+                setScheduledChangeOperation: (state, { changeType }) =>
+                    changeType === ScheduledChangeOperationType.UpdateStatus ? state : false,
+            },
+        ],
+        recurrenceInterval: [
+            null as RecurrenceInterval | null,
+            {
+                setRecurrenceInterval: (_, { interval }) => interval,
+                // Reset when recurring is disabled or operation changes
+                setIsRecurring: (state, { isRecurring }) => (isRecurring ? state : null),
```

Comment:
when `isRecurring` is set to false, `recurrenceInterval` is reset to null, but this conflicts with how we "pause" execution in the backend where `recurrence_interval` is preserved when `is_recurring` is false. If a user creates a recurring schedule, pauses it, then tries to create a new non-recurring schedule, the old interval might leak through. The frontend should distinguish between "creating new" vs "pausing existing."